pkg/apis/build/v1alpha1/notary_types.go

```diff
@@ -0,0 +1,24 @@
+package v1alpha1
+
+// +k8s:openapi-gen=true
+type NotaryConfig struct {
+	V1 *NotaryV1Config `json:"v1,omitempty"`
+}
+
+// +k8s:openapi-gen=true
+type NotaryV1Config struct {
+	URL             string           `json:"url"`
+	SecretRef       NotarySecretRef  `json:"secretRef"`
+	ConfigMapKeyRef *ConfigMapKeyRef `json:"configMapKeyRef,omitempty"`
+}
+
+// +k8s:openapi-gen=true
+type NotarySecretRef struct {
+	Name string `json:"name"`
+}
+
+// +k8s:openapi-gen=true
+type ConfigMapKeyRef struct {
```

Comment:
Sorry, if you already outlined this but, could you explain how this cert is used again?

Reply:
This is the certificate for the notary server. For harbor, that is usually the same cert as the registry, but that may not always be the case. If we drop the ability to provide the cert here, then users have no way of providing a cert without modifying the completion image that we provide (unless they had some kind of cert injection mechanism).

Reply:
How is that different from the current ability for users to provide a cert for their registry?

Reply:
Removed as it was inconsistent with how registry certs are provided.